packages/terra-table/src/TableHeader.module.scss

```diff
@@ -0,0 +1,9 @@
+/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
```

Comment:
I think we can remove the stylelint-disable line here

Reply:
Updated.